samples/todolist-aspnetcore/README.md

```diff
@@ -0,0 +1,17 @@
+# ASP.NET Core API To Do List Sample
+
+This sample is an ASP.NET Core app that calls a Durable Function in an existing function app. The web application is a list of To Do list tasks and each time a task is added, a SetReminder function is called that schedules a reminder text to be sent to your phone after 24 hours to remind you to complete the task. The SetReminder orchestrator function is a function in an existing function app. The way the web application and function app are able to communicate is by having the same storage account and task hub.�
```

Comment:
Small style nits:

Instead of "The SetReminder orchestrator function is a function in an existing function app.", include a link to the actual function in the precompiled sample project.

Instead of "The way the web application and function app are able to communicate is by having the same storage account and task hub.", consider "The provided storage account connection string and task hub name for this application must match the connection string and task hub name used for the function app."